index.html

```diff
@@ -64,5 +66,182 @@ <h1>I’m MERCY <br />Glad to see <br />you!</h1>
+        </div>
+        <p class="cards-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="grid-skills">
+          <div class="item1">html</div>
+          <div class="item2">css</div>
+          <div class="item3">javascript</div>
+        </div>
+        <button id="popup">see project</button>
+      </div>
+    </section>
+    <!--card3-->
+    <section class="projects-section" id="section-cards">
+      <div class="cards">
+        <img src="/assets/images/project3.png" alt="project1" />
+      </div>
+      <div class="cards-details">
+        <div>
+        <h2 class="title">Tonic</h2>
+        <div class="highlights">
+          <p class="highlight-text">Canopy</p>
+            <img class="counter" src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">Back End Dev</p>
+            <img src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">2015</p>
+        </div>
+        </div>
+        <p class="cards-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="grid-skills">
+          <div class="item1">html</div>
+          <div class="item2">css</div>
+          <div class="item3">javascript</div>
+        </div>
+        <button id="popup">see project</button>
+      </div>
+    </section>
+    <!--card4-->
+    <section class="projects-section" id="section-cards">
+      <div class="cards">
+        <img src="/assets/images/project4.png" alt="project1" />
+      </div>
+      <div class="cards-details">
+        <div>
+        <h2 class="title">Tonic</h2>
+        <div class="highlights">
+          <p class="highlight-text">Canopy</p>
+            <img class="counter" src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">Back End Dev</p>
+            <img src="/assets/images/Counter.png" alt="Counter" />
+            <p class="countertext">2015</p>
+        </div>
+        </div>
+        <p class="cards-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <div class="grid-skills">
+          <div class="item1">html</div>
+          <div class="item2">css</div>
+          <div class="item3">javascript</div>
+        </div>
+        <button id="popup">see project</button>
+      </div>
+    </section>
+    <!--About me-->
+    <section class="heading1" id="about">
+      <div class="header">
+        <h1>I’m MERCY</h1>
+      </div>
+      <div class="heading-paragraph">
+        <p>
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
```

Comment:
Hey 👋🏽 please use the right content and the right design here. Your `About` section doesn't fit the Figma design. The main differences are the spacing and the rounded corner of the About section. See below 👇🏽 

## Figma
![Screenshot 2023-02-17 at 21 23 41](https://user-images.githubusercontent.com/45739963/219826859-a553ea03-3481-4735-8d6d-f638625271fe.png)

## Your design
![Screenshot 2023-02-17 at 21 24 13](https://user-images.githubusercontent.com/45739963/219826877-d72ba1c6-be28-4238-a934-5342f25733c0.png)